src/util/BrowserFeatures.js

```diff
@@ -68,6 +68,12 @@ define(function () {
     return navigator.platform.toUpperCase().indexOf('MAC')>=0;
   };
 
+  fn.isMobileDevice = function () {
+    // relying on UA requires very extensive checks.
+    // Using this based on https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation.
```

Comment:
the feature is deprecated and no support at desktop. any better solutions?

Reply:
Changes it to just check ios or androind since OV is just available in those platforms.